showcase-docs/getting-started.md

```diff
@@ -32,7 +32,16 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
    - If you are on Windows, then follow the [instructions](https://github.com/nodejs/node-gyp#on-windows) in `node-gyp` for Windows
    - If you are on macOS, then follow the [instructions](https://github.com/nodejs/node-gyp#on-macos) in `node-gyp` for macOS
 
-3. Copy the required code snippet from `app-config.yaml` into `app-config.local.yaml`. Note: Each plugin has a `# Plugin: <PLUGIN_NAME>` comment above the required code snippet(s).
+3. Run `yarn install` to install the dependencies
+
+4. Make sure you that, in the `dynamic-plugins-root` folder, you have the dynamic plugins you want to load in the backend application. To have all the plugins normally included in the Showcase application,
```

Comment:
```suggestion
4. Make sure you that, in the `dynamic-plugins-root` folder, you have the dynamic plugins you want to load in the backend application. To have all the plugins originally included in the Showcase application,
```
As an additional suggestion to the suggestion above https://github.com/janus-idp/backstage-showcase/pull/646/files#r1368979215